nemo/collections/common/tokenizers/sentencepiece_tokenizer.py

```diff
@@ -30,7 +31,9 @@ class SentencePieceTokenizer(TokenizerSpec):
     Sentencepiecetokenizer https://github.com/google/sentencepiece.
     '''
 
-    def __init__(self, model_path: str, special_tokens: Optional[Union[Dict[str, str], List[str]]] = None):
+    def __init__(
+        self, model_path: str, special_tokens: Optional[Union[Dict[str, str], List[str]]] = None, legacy: bool = False
+    ):
         """
         Args:
             model_path: path to sentence piece tokenizer model. To create the model use create_spt_model()
```

Comment:
can you please add "legacy" to docstring. Say something brief.

Reply:
Addressed both comments. Waiting for @yzhang123 's review. Will merge once approved.

Reply:
could you move docstring outside of constructor for automatic doc generation?

Reply:
Moved the args docstring outside. Is it fine now?